geps/gep-2257.md

```diff
@@ -57,6 +57,16 @@ Durations are a subset of what `time.ParseDuration` supports:
   not a valid GEP-2257 duration string (though `0s` is). Likewise the empty
   string is not a valid GEP-2257 duration.
 
+- Users SHOULD represent the zero duration as `0s`, although they MAY use any
+  of `0h`, `0m`, etc. Implementations formatting a GEP-2257 Duration for
+  output MUST render the zero duration as `0s`.
```

Comment:
one nit: Kube APIs MUST NOT modify user input, even to normalize.  If the user's `spec` says "29m60s1h" then the spec stays that way.